### ISSUE TITLE
Doesn't load for Rails 4.2

### ISSUE THREAD

**Issue by borourke** [shideneyu/kraken_client#6]
These are the first few lines of my gemfile:

```
source 'https://rubygems.org'

gem 'kraken_client'

# Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
gem 'rails', '4.2.6'
# Use sqlite3 as the database for Active Record
gem 'sqlite3'
# Use SCSS for stylesheets
gem 'sass-rails', '~> 5.0'
# Use Uglifier as compressor for JavaScript assets
gem 'uglifier', '>= 1.3.0'
# Use CoffeeScript for .coffee assets and views
gem 'coffee-rails', '~> 4.1.0'
```

Run bundle install, says it installs, but then class can't be found:

```
borourke$ bundle exec rails c
Running via Spring preloader in process 79949
Loading development environment (Rails 4.2.6)
irb(main):001:0> KrakenClient.load({
irb(main):002:2*       api_key: XXXXXXXX,
irb(main):003:2*       api_secret: XXXXXX
irb(main):004:2>     })
NameError: uninitialized constant KrakenClient
    from (irb):1
    from /Users/borourke/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/railties-4.2.6/lib/rails/commands/console.rb:110:in `start'
    from /Users/borourke/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/railties-4.2.6/lib/rails/commands/console.rb:9:in `start'
    from /Users/borourke/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/railties-4.2.6/lib/rails/commands/commands_tasks.rb:68:in `console'
    from /Users/borourke/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/railties-4.2.6/lib/rails/commands/commands_tasks.rb:39:in `run_command!'
    from /Users/borourke/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/railties-4.2.6/lib/rails/commands.rb:17:in `<top (required)>'
    from /Users/borourke/Projects/ktifu/bin/rails:9:in `<top (required)>'
    from /Users/borourke/.rbenv/versions/2.2.3/lib/ruby/2.2.0/rubygems/core_ext/kernel_require.rb:54:in `require'
    from /Users/borourke/.rbenv/versions/2.2.3/lib/ruby/2.2.0/rubygems/core_ext/kernel_require.rb:54:in `require'
    from -e:1:in `<main>'
irb(main):005:0>
```

I can use the class in a script, but not in a gemfile/rails context.  Would be really nice if it supported that.

**Comment by shideneyu** [comment]
Hello, thanks you for your report.

This is fixed with kraken client 1.1.5. Please update your gemfile accordingly and tell me if this works for you.

**Comment by borourke** [comment]
Works great.  Thanks for responding so quickly and effectively.
